woocommerce/wc-admin
# Make sure the depreciated onClick is triggered and container is used

Fixes issue where old extended task lists `onClick` wasn't called, or the `container` param wasn't used.

No changelog necessary

### Detailed test instructions:

- Load this branch and install the Google Listings & Ads plugin ( available under the marketing tools task)
- Notice the new task list (Things to do next) click on `Set up Google Listings & ads`, it should correctly redirect to the plugins set up screen.
- Run `npm run example -- --ext=add-task` locally and enable the **WooCommerce Admin add task** plugin
- Click on the **Example** task displayed in the extended task list, it should render the container with the **Mark task complete** button.
- Add another task with a different `key` to [the example](https://github.com/woocommerce/woocommerce-admin/blob/main/docs/examples/extensions/add-task/js/index.js) with `level: 1` and make sure it shows up at the top of the list.
- Then update the Example task, setting `completed` to `true`, and make sure it's displayed at the bottom of the list now.
- The main task list should not be affected by this sorting.
